optaplanner-examples/src/main/java/org/optaplanner/examples/cheaptime/solver/drools/MachinePeriodPart.java

```diff
@@ -97,40 +104,30 @@ public int getPeriod() {
     }
 
     @Override
-    public boolean equals(Object o) {
+    public boolean equals(final Object o) {
         if (this == o) {
             return true;
-        } else if (o instanceof MachinePeriodPart) {
-            MachinePeriodPart other = (MachinePeriodPart) o;
-            return new EqualsBuilder()
-                    .append(machine, other.machine)
-                    .append(period, other.period)
-                    .append(active, other.active)
-                    .append(resourceAvailableList, other.resourceAvailableList)
-                    .isEquals();
-        } else {
+        }
+        if (o == null || getClass() != o.getClass()) {
             return false;
         }
+        final MachinePeriodPart that = (MachinePeriodPart) o;
+        return period == that.period &&
+                active == that.active &&
+                Objects.equals(machine, that.machine) &&
```

Comment:
Code style: please keep original order (machine, period, active), which is the order of the field declarations.